runtime/compiler/CMakeLists.txt

```diff
@@ -154,6 +154,7 @@ endif()
 # Some files in OMR provide duplicate or extra functionality not needed
 # in J9, so we need to remove them
 set(REMOVED_OMR_FILES
+	${omr_SOURCE_DIR}/compiler/codegen/CCData.cpp
```

Comment:
Is there a problem compiling it on z/OS? I do understand it is not being used (currently).

Reply:
There is an issue with some of the macros defined by the a2e headers conflicting with system headers. IIRC the issue was the system `xlocale` using names which clashed with macros defined in the a2e version of `ctype.h`

Reply:
Ok thanks. We'll leave this comment unresolved so we can come back to this when we need to compile that file. To make forward progress though I'm ok to merge.